public/app/plugins/panel/state-timeline/StateTimelinePanel.tsx

```diff
@@ -1,9 +1,18 @@
-import React, { useMemo } from 'react';
-import { PanelProps } from '@grafana/data';
-import { useTheme2, ZoomPlugin } from '@grafana/ui';
+import React, { useCallback, useMemo } from 'react';
+import {
+  DataFrame,
+  FALLBACK_COLOR,
+  formattedValueToString,
+  getDisplayProcessor,
+  getFieldDisplayName,
+  PanelProps,
+} from '@grafana/data';
+import { TooltipDisplayMode, TooltipPlugin, useTheme2, ZoomPlugin } from '@grafana/ui';
 import { TimelineMode, TimelineOptions } from './types';
 import { TimelineChart } from './TimelineChart';
-import { prepareTimelineFields, prepareTimelineLegendItems } from './utils';
+import { findNextStateIndex, prepareTimelineFields, prepareTimelineLegendItems } from './utils';
+import { Interval, toDuration } from '@grafana/data/src/valueFormats/dateTimeFormatters';
```

Comment:
you do not need to import this, you can import getValueFormat and then call getValueFormat('dtdurationms');   get the same formatter you are using below

Reply:
Ah, awesome! Thanks @torkelo